service/src/main/java/uk/gov/hmcts/reform/fpl/service/respondent/RespondentValidator.java

```diff
@@ -35,11 +35,15 @@ public class RespondentValidator {
     private final Time time;
 
     public List<String> validate(CaseData caseData, CaseData caseDataBefore) {
+        return validate(caseData, caseDataBefore, false);
+    }
+
+    public List<String> validate(CaseData caseData, CaseData caseDataBefore, boolean hideRespondentIndex) {
         List<String> errors = new ArrayList<>();
 
         validateMaximumSize(caseData).ifPresent(errors::add);
 
-        errors.addAll(validateDob(caseData));
+        errors.addAll(validateDob(caseData, hideRespondentIndex));
         errors.addAll(validateAddress(caseData));
```

Comment:
do we need a hideRespondentIndex here?

Reply:
No, we don't need it. Since the message in validateAddress does not contain the information of the respondent's index.